core/src/main/java/edu/wpi/grip/core/serialization/Project.java

```diff
@@ -20,7 +22,8 @@
 import java.io.Writer;
 import java.nio.charset.StandardCharsets;
 import java.util.Optional;
-
+import javafx.beans.property.SimpleBooleanProperty;
+import javafx.beans.value.ChangeListener;
```

Comment:
JavaFX is bundled in the desktop JRE, so it compiles fine. Project Jigsaw solves this, but it's getting released in March...